Src/Support/Google.Apis.Auth/OAuth2/AccessTokenCredential.cs

```diff
@@ -42,22 +43,24 @@ internal class AccessTokenCredential : IGoogleCredential, IHttpExecuteIntercepto
         /// <inheritdoc/>
         bool IGoogleCredential.SupportsExplicitScopes => false;
 
-        public AccessTokenCredential(string accessToken, IAccessMethod accessMethod, string quotaProject = null)
+        public AccessTokenCredential(
```

Comment:
This is a (binary) breaking change - we need to add an overload without any optional parameters, I suspect. (Or add a private constructor with all the parameters, and a static factory method.)